assets/js/components/setup/compatibility-checks.js

```diff
@@ -142,9 +168,45 @@ export default class CompatibilityChecks extends Component {
 						external={ external }
 						inherit
 					/>
-				</Fragment>;
+				</p>;
 			case ERROR_TOKEN_MISMATCH:
-				return __( 'Looks like you may be using a caching plugin which could interfere with setup. Please deactivate any caching plugins before setting up Site Kit. You may reactivate them once setup has been completed.', 'google-site-kit' );
+				return <p>{ __( 'Looks like you may be using a caching plugin which could interfere with setup. Please deactivate any caching plugins before setting up Site Kit. You may reactivate them once setup has been completed.', 'google-site-kit' ) }</p>;
+			case ERROR_GOOGLE_API_CONNECTION_FAIL:
+				return <Fragment>
+					<p
+						dangerouslySetInnerHTML={ sanitizeHTML(
+							sprintf(
+							/* translators: %s: Support Forum URL */
+								__( 'Looks like your site is having a technical issue with requesting data from Google services.<br/>To get more help, ask a question on our <a href="%s">support forum</a> and include the text of the original error message:<br/>', 'google-site-kit' ),
+								'https://wordpress.org/support/plugin/google-site-kit/'
+							),
+							{
+								ALLOWED_TAGS: [ 'a', 'br' ],
+								ALLOWED_ATTR: [ 'href' ],
+							}
+						) }
+
+					/>
+					<p>{ error }</p>
```

Comment:
The error should be included in the translated string using a placeholder with `sprintf`. As it is, this is concatenating the message with the error, which is not i18n-friendly.